frappe_telegram/frappe_telegram/doctype/telegram_message_template_translation/telegram_message_template_translation.json

```diff
@@ -0,0 +1,41 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2021-11-15 20:47:24.219471",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "language",
+  "template"
+ ],
+ "fields": [
+  {
+   "fieldname": "language",
+   "fieldtype": "Data",
```

Comment:
@Abadulrehman Why not `Link` to `Language` ?

Reply:
Hmm, you're right. I was evening considering making it a link, but then did not due to misunderstanding.